kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeStagingDataTab.js

```diff
@@ -84,9 +84,9 @@ define([
                     userInfo: userInfo,
                     userId: Jupyter.narrative.userId
                 });
-                this.uploadWidget.dropzone.on('complete', function() {
-                    this.updateView();
-                }.bind(this));
+                // this.uploadWidget.dropzone.on('complete', function() {
```

Comment:
Is this supposed to still be commented out?

Reply:
Nope, that should be deleted. Thanks for catching.